### PR TITLE
manage service module

### PR DESCRIPTION
**Pull request by ShrutiSachpara**

### CODE REVIEW

File: src/app/service-manage/service-manage.component.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+  submitForm() {
+    if (this.serviceForm.valid) {
+      const formData = this.serviceForm.value;
+      if (this.isEditMode) {
+        this.updateService(formData);
+      } else {
+        this.getService(formData);
+      }
+    } else {
```
Comment (krishna2919) [inline src/app/service-manage/service-manage.component.ts:199]: remove this console and else

File: src/app/service-manage/service-manage.component.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+import Swal from 'sweetalert2';
+import { Messages } from 'src/helper/message';
+
+@Component({
+  selector: 'app-service-manage',
+  templateUrl: './service-manage.component.html',
+  styleUrls: ['./service-manage.component.css'],
+})
+export class ServiceManageComponent {
+  listOfManageServiceData: any[] = [];
```
Comment (SwetaaRampariya) [inline src/app/service-manage/service-manage.component.ts:15]: do not use any type anywhere

File: src/app/ngbdSortableHeader.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    '[class.desc]': 'direction === "desc"',
+  },
+})
+export class NgbdSortableHeader {
+  @Input() sortable: SortColumn = '';
+  @Input() direction: SortDirection = '';
+  @Output() sort = new EventEmitter<SortEvent>();
+
+  @HostListener('click', ['$event']) rotate() {
+    console.info('clicked: ');
```
Comment (SwetaaRampariya) [inline src/app/ngbdSortableHeader.ts:34]: remove consoles.

File: src/app/service-manage/service-manage.component.ts [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    if (selectedEvent) {
+      this.isEditMode = true;
+      this.selectedServiceId = data.id;
+      this.serviceForm.patchValue(selectedEvent);
+      this.openModal();
+    }
+  }
+
+  deleteEvent(data: any) {
+    Swal.fire({
```
Comment (SwetaaRampariya) [inline src/app/service-manage/service-manage.component.ts:151]: use comman file

**Pull request merged**